test_vm/tests/suite/test_vm_test.rs

```diff
@@ -22,25 +20,14 @@ fn state_control() {
     let addr2 = Address::new_id(2222);
 
     // set actor
-    let a1 = new_actor(
-        *ACCOUNT_ACTOR_CODE_ID,
-        make_identity_cid(b"a1-head"),
-        42,
-        TokenAmount::from_atto(10u8),
-        None,
-    );
+    let a1 =
```

Comment:
these changes are just for expedience, right? nothing else going on here other than it's just easier to pull in an existing CID than make one with the new libraries?

Reply:
Yes. These needed to be valid CBOR CIDs and that was the first one I reached for.